src/Nethermind/Nethermind.Test.Runner/Program.cs

```diff
@@ -2,9 +2,10 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 
 using System;
+using System.CommandLine;
 using System.IO;
+using System.Threading;
 using System.Threading.Tasks;
-using CommandLine;
```

Comment:
The `CommandLineParser` package has been removed and replaced with `System.CommandLine`. Regardless of what parser we use, it's better to use the same everywhere for more effective maintenance. No other changes in the logic.